### DISCUSSION TITLE
Support for Adobe Animate Atlas

### DISCUSSION THREAD

**Discussion by animald** [pixijs/pixijs#7688] [General]
I was disheartened to find a stale issue regarding this from 2018... #4273

It's now 2021, please tell me someone has cracked this and found a way to use Adobe Animate atlas animations within Pixi?

**Comment by miltoncandelero** [comment]
I think the biggest problem is that there is no docs on how the json works. I even remember Adobe changing the name of the keys in the json to one letter in order to "minify" the json 🤔🤦‍♀️

**Comment by bigtimebuddy** [comment]
We have our own support for Animate with a custom extension. This is not Atlas, but will allow you to export content for Pixi.

Runtime: https://github.com/pixijs/pixi-animate
Animate Extension: https://github.com/pixijs/pixi-animate-extension

**Reply by joao-cesar** [reply]
Hi, @bigtimebuddy!

I use Adobe Animate on a daily basis and I also provide help in the official Adobe Forums.

I've been asking Adobe for GPU acceleration in the HTML5 Canvas output (basically enabling StageGL in CreateJS) but it seems that the priority for the Animate team now is to improve the animation tools and features.

So if you don't mind I would like to ask if the extension is ready for production and if you plan to continue supporting it.

Just today I realized that the official GitHub of Pixi Animate was updated recently so I started running some tests and I got very excited about the possibilities of using such a powerful rendering system like PixiJS.
